src-java/floodlight-service/floodlight-api/src/main/java/org/openkilda/messaging/info/stats/GroupStatsEntry.java

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.messaging.info.stats;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.Value;
+
+import java.io.Serializable;
+
+@Value
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+public class GroupStatsEntry implements Serializable {
+
+    @JsonProperty
+    int groupId;
+
+    @JsonProperty
+    long byteInCount;
+
+    @JsonProperty
+    long packetsInCount;
```

Comment:
Minor: if you are using `@JsonNaming` you can remove all `@JsonProperty`

Reply:
thanks, fixed